frankDoc/src/test/java/nl/nn/adapterframework/frankdoc/model/DigesterRulesPatternTest.java

```diff
@@ -0,0 +1,113 @@
+package nl.nn.adapterframework.frankdoc.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.junit.Test;
+
+public class DigesterRulesPatternTest {
+	@Test
+	public void whenPatternIsNullThenInvalid() {
+		assertNotNull(new DigesterRulesPattern(null).getError());
+	}
+
+	@Test
+	public void whenPatternIsEmptyThenInvalid() {
+		assertNotNull(new DigesterRulesPattern("").getError());
+	}
+
+	@Test
+	public void whenPatternHasMultipleWordsWithoutWildcardThenNotSupported() {
+		assertNotNull(new DigesterRulesPattern("first/second").getError());
```

Comment:
Dit is toch een valide pattern? De wildcards worden gebruikt om het leesbaarder te maken, niet omdat ze er in moeten. Het zou ook mogelijk moeten zijn om configuration/adapter/pipeline/pipe te gebruiken (multi-level). Wat is de reden dat nu eerst alles ingeladen wordt en daarna een laag filters er over heen gaan?